MODS.js

```diff
@@ -372,6 +372,10 @@ function doExport() {
 		
 		/** CORE FIELDS **/
 		
+		recordInfo = doc.createElementNS(ns, "recordInfo");
```

Comment:
That is already there some lines above https://github.com/zotero/translators/blob/5a023542b96a4fc915636ed073c059a398ab2d3e/MODS.js#L368